src/test/resources/expectedOutput/search_spatial_explain.json

```diff
@@ -12,7 +12,6 @@
                                     "place" : {
                                         "shape" : {
                                             "type" : "polygon",
-                                            "orientation" : "right",
```

Comment:
Is this a breaking change in ES 7.4.2 DSL?

Reply:
Yes, after the compatibility changes are made for ES 7.4.2 that element no longer is included, and the IT will fail if it is left in there.

Reply:
Got it. Thanks!